common/djangoapps/course_category/models.py

```diff
@@ -1,31 +1,40 @@
+import django.dispatch
 from django.db import models
-from django.db.models.signals import post_delete, post_save
+from django.db.models.signals import m2m_changed
 from django.dispatch import Signal, receiver
 from django.utils.translation import ugettext_lazy as _
 
 from mptt.models import MPTTModel, TreeForeignKey
 from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
 
-from .tasks import task_reindex_course, task_reindex_courses
+from .tasks import task_reindex_courses
+
+move_to = django.dispatch.Signal()
 
 
 class CourseCategory(MPTTModel):
     name = models.CharField(max_length=255, verbose_name=_("Category Name"), unique=True)
     description = models.TextField(null=True, blank=True)
+    img = models.ImageField(upload_to='course_category', blank=True)
     parent = TreeForeignKey('self', null=True, blank=True, related_name='children', db_index=True)
     enabled = models.BooleanField(default=True)
     slug = models.SlugField(max_length=255, unique=True)
+    url = models.URLField(max_length=200, null=True, blank=True)
+    courses = models.ManyToManyField("course_overviews.CourseOverview", blank=True)
```

Comment:
`blank=True` doesn't affect the DB field, it's True by default, moreover you have set `required=False` in form